tests/accel/upgrade/workload.go

```diff
@@ -0,0 +1,215 @@
+package upgrade
```

Comment:
Please move it to ./tests/accel/upgrade/tests directory

Reply:
Done, thanks